src/main/java/com/bakdata/kafka/KafkaProducerApplication.java

```diff
@@ -126,8 +145,15 @@ protected void cleanUpRun(final SchemaTopicClient schemaTopicClient) {
         }
     }
 
+    private static void deleteTopicAndResetSchema(final TopicClient topicClient,
```

Comment:
Do not duplicate code here. Beforehand it was nice and reusable. Probably related to my comment above